components/Login/Login.tsx

```diff
@@ -29,23 +29,20 @@ const Login: React.FC = () => {
         token: string;
       };
     }) => {
-      setError('');
       dispatch(logIn(user.data));
-      // Navigate to the wizard
+      navigation.navigate(Routes.FitnessWizard);
     },
   });
 
-  const [error, setError] = useState('');
-
-  const buttonType: ButtonType = 'submit';
-  const keyBoardType: KeyboardTypeOptions = 'email-address';
+  const type: ButtonType = 'submit';
+  const keyboardType: KeyboardTypeOptions = 'email-address';
```

Comment:
There is a lot of duplication with these constants, you can use `enum` or global constants to fix this.